backend/app/assets/stylesheets/spree/backend/components/_navigation.scss

```diff
@@ -24,13 +28,12 @@ nav.menu {
 }
 
 .admin-nav {
-  border-right: 1px solid $color-sidebar-border;
   background: $color-sidebar-bg;
   z-index: $z-index-navbar-flyout;
 }
 
 .admin-nav-menu {
-  text-transform: uppercase;
+  text-transform: capitalize;
```

Comment:
Ha! Good point. I think I wanted `uppercase` gone, and saw `capitalize` as the next step down. Let's be rid of it entirely!